backends/cuda-gen/ceed-cuda-gen-qfunction.c

```diff
@@ -107,7 +107,7 @@ int CeedQFunctionCreate_Cuda_gen(CeedQFunction qf) {
 
   char *source;
   ierr = CeedQFunctionGetSourcePath(qf, &source); CeedChk(ierr);
-  const char *funname = strrchr(source, ':') + 1;
+  cons char *funname = strrchr(source, ':') + 1;
```

Comment:
?
This was clearly missed by testing.

Reply:
Woops, I'll fix it in a small PR here in a bit